iModelCore/ECPresentation/Source/Content/ContentItemBuilder.cpp

```diff
@@ -25,7 +25,11 @@ rapidjson::Document ContentValuesFormatter::GetFallbackPrimitiveValue(ECProperty
     rapidjson::Document json(allocator);
     ECValue v = ValueHelpers::GetECValueFromSqlValue(type, extendedType, value);
     Utf8String stringValue;
-    if (v.ConvertPrimitiveToString(stringValue))
+    if ((type == PRIMITIVETYPE_Point3d || type == PRIMITIVETYPE_Point2d) && v.IsNull())

```

Comment:
Why is this needed? Does `ConvertPrimitiveToString` return `false` if `ECValue` is `null`?